tests/std/tests/P0355R7_calendars_and_time_zones_io/test.cpp

```diff
@@ -888,22 +892,68 @@ void insert_leap_second(const sys_days& date, const seconds& value) {
         tzdb{my_tzdb.version, move(zones), move(links), move(leap_vec), my_tzdb._All_ls_positive && (value == 1s)});
 }
 
+void test_gh_1952() {
+    auto time_str{"2021-06-02T17:51:05.696028Z"};
+    auto fmt{"%FT%TZ"};
+    auto utc_ref = clock_cast<utc_clock>(sys_days{2021y / June / 2d} + 17h + 51min + 05s + 696028us);

```

Comment:
I think we should avoid the leading zero in `05s` - although it mirrors `time_str`, it's an octal UDL, which is weird (and doesn't appear in other test code).

Reply:
An octal literal, good catch.